.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before submitting a bug report**
+
+        The issue list is reserved for bug reports and feature requests. If you have a usage question, you can:
+
+        - Read the [documentation](https://blazorise.com/docs)
+        - Ask questions on [Discord](https://discord.io/blazorise)
+
+        Also, try to search for your issue/feature - another user may have already requested something similar; Thanks!
+
+  - type: input
+    id: version
+    attributes:
+      label: Blazorise Version
+    validations:
+      required: true
+  - type: input
+    id: reproduction-link
+    attributes:
+      label: Link to minimal reproduction
+      description: |
+        The easiest way to provide a reproduction is to provide a link to a public GitHub repository or a tool like [Telerik Repl](https://blazorrepl.telerik.com/).
+
+        The reproduction should be **minimal**. This means it should contain only the bare minimum amount of code needed
+        to show the bug.
+      placeholder: Reproduction Link
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        What do we need to do after opening your repro to make the bug happen? Clear and concise reproduction instructions are important for us to be able to triage your issue in a timely manner. Note that you can use [Markdown](https://guides.github.com/features/mastering-markdown/) to format lists and code.
```

Comment:
Maybe link to the lists and code anchors, instead of full documention?
i.e https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists